docs/source/executing-operations/native-query-planner.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Native Query Planner
+subtitle: Run the Rust-native query planner in GraphOS Router
+minVersion: 1.49.0
+---
+
+<PreviewFeature />
+
+Learn to run the GraphOS Router with the Rust-native query planner and improve your query planning performance and scalability.
+
+## Background about query planner implementations
+
+In v1.49.0 the router introduced a [query planner](/graphos/routing/about-router#query-planning) implemented natively in Rust. This native query planner improves the overall performance and resource utilization of query planning. It exists alongside the legacy Javascript implementation. and it will eventually replace the legacy implementation.
```

Comment:
```suggestion
In v1.49.0 the router introduced a [query planner](/graphos/routing/about-router#query-planning) implemented natively in Rust. This native query planner improves the overall performance and resource utilization of query planning. It exists alongside the legacy Javascript implementation, and it will eventually replace the legacy implementation.
```